rotkehlchen/globaldb/handler.py

```diff
@@ -627,12 +627,19 @@ def get_asset_data(
     def fetch_underlying_tokens(
             cursor: DBCursor,
             parent_token_identifier: str,
+            from_builtin_db: bool = False,
     ) -> Optional[List[UnderlyingToken]]:
         """Fetch underlying tokens for a token address if they exist"""
-        cursor.execute(
-            'SELECT B.address, B.token_kind, A.weight FROM underlying_tokens_list AS A JOIN evm_tokens as B WHERE A.identifier=B.identifier AND parent_token_entry=?;',  # noqa: E501
-            (parent_token_identifier,),
-        )
+        if from_builtin_db is True:
+            cursor.execute(
+                'SELECT B.address, B.token_kind, A.weight FROM builtin_db.underlying_tokens_list AS A JOIN builtin_db.evm_tokens as B WHERE A.identifier=B.identifier AND parent_token_entry=?;',  # noqa: E501
+                (parent_token_identifier,),
+            )
+        else:
+            cursor.execute(
+                'SELECT B.address, B.token_kind, A.weight FROM underlying_tokens_list AS A JOIN evm_tokens as B WHERE A.identifier=B.identifier AND parent_token_entry=?;',  # noqa: E501
+                (parent_token_identifier,),
+            )
```

Comment:
I believe you don't need to attach it. Just changing the cursor you use should be enough. The performance of the database can be affected.

edit: I searched and found this comment regarding the topic https://stackoverflow.com/a/50782262